.github/workflows/integration-tests.yml

```
@@ -9,6 +9,32 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
+  release-package-sanity:
+    runs-on: ubuntu-latest
+
+    name: Release package sanity test
+
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)